packages/components/select-menu/src/select-menu.js

```diff
@@ -122,25 +129,40 @@ export class TSSelectMenu extends TSElement {
 		</div>`;
 	}
 
+	get isVisibleShowSelectedButton() {
+		return this.multiselect && (this.dirty || this.currentSelection.length);
+	}
+
 	/** @private */
 	get selectButton() {
 		return html`
-			<ts-button-group>
+			<ts-button-group class="apply-button-container ${this.showApplyButtonContainer ? 'show' : 'hide'}">
 				<ts-button type="primary" @click=${this.applySelection}
 					>${this.translations.select} ${this.currentSelection.length}</ts-button
 				>
```

Comment:
Strange line break in a closing tag (I assume it is from prettier). Please, fix.